client/src/components/editor/LinkSource.js

```diff
@@ -46,7 +46,7 @@ class LinkSource extends Component {
     if (url) {
       const selection = editorState.getSelection()
       const entityData = {
-        url: url.replace(/\s/g, "")
+        url: url
```

Comment:
Why this change (or: why were spacced stripped before)?

Reply:
This comment applies to a file no longer in use.